src/pages/Pokedex/layout.tsx

```diff
@@ -4,17 +4,12 @@ import { ReactNode } from "react";
 
 interface BasicCardProps {
   children?: ReactNode;
+  loadPrevious: () => void;
+  loadNext: () => void;
+  offset: number;
 }
 
-export default function BasicCard({ children }: Readonly<BasicCardProps>) {
-  const handleNext = () => {
-    console.log("Handle next button click");
-  };
-
-  const handlePrevious = () => {
-    console.log("Handle previous button click");
-  };
-
+export default function BasicCard({ children, loadPrevious, loadNext, offset }: Readonly<BasicCardProps>) {
```

Comment:
Consider marking children as required (ReactNode) in the interface definition if there are cases where the component would not function without it.